Update PROS.svg to fix display issue on iOS / Safari

prosper-fi opened the pull request:
Hi team,

Just a minor update to a parameter on the SVG that was causing it to show as grayscale on iOS and Safari only (was fine on Chrome).

Thank you!

MRabenda commented:
@prosper-fi as we introduced mutlichain repo structure - can you update your icon in https://github.com/MetaMask/contract-metadata/tree/master/icons/eip155%3A1 folder as well?

MRabenda commented:
Thanks,

I see that you would like also add BSC version - in this case there sould be also new icon in folder of bsc icons like icons/eip155:56/erc20:0x915424Ac489433130d92B04096F3b96c82e92a9D.svg

prosper-fi commented:
Hi @MRabenda , thank you for the feedback! We've:

1. Updated the SVG in the ETH folder
2. Added the SVG to the BSC icons folder
3. Added a JSON to the BSC metadata folder

Will you guys have a similar structure for Base in the future too?

Let us know if there's anything else we should update. Thank you!

MRabenda commented:
You can add Base today - chain ID is 8453 - we are supporting any chain today, so you can create folders yourself

prosper-fi commented:
Ah okay, let us go ahead and do that then!

MRabenda commented:
Thanks @prosper-fi for you contribution!
